### PR TITLE
Add feature and choose your own adventure projects

### PR DESCRIPTION
**Pull request by annajung**
### Description

- Add featured and choose your own adventure projects

### CODE REVIEW

File: _pages/open-source-day-2021-summer-contribution.md [hunk 1]
```diff
@@ -13,13 +13,27 @@ As part of the Contribution Track, you will contribute code to curated open sour
...
+| [Greenpeace Planet 4](https://github.com/greenpeace) | Built on top of Wordpress tech, Greenpeace Planet 4 powers digital platforms to engage with millions and win campaigns around the world. | [Suzi Grishpul](https://github.com/suzi-greenpeace) |
+| [Material-UI](https://github.com/mui-org/material-ui) | Material-UI is a simple and customizable component library to build faster, beautiful, and more accessible React applications. Follow your own design system, or start with Material Design. | [Marija Najdova](https://github.com/mnajdova) |
+| [Openstack](https://opendev.org/openstack) | OpenStack is a cloud operating system that controls large pools of compute, storage, and networking resources throughout a datacenter, all managed through a dashboard that gives administrators control while empowering their users to provision resources through a web interface. | [Amy Marrich]() |
```
Comment (annajung) [inline _pages/open-source-day-2021-summer-contribution.md:22]: Note: Unable to locate Amy's link, will need to reach out

File: _pages/open-source-day-2021-summer-contribution.md [hunk 2]
```diff
@@ -13,13 +13,27 @@ As part of the Contribution Track, you will contribute code to curated open sour
...
+| [OpenStreetMap](https://github.com/Intuit/Superglue) | TBA | TBA |
+| [OpenSUSE](https://github.com/openSUSE) | TBA | TBA |
```
Comment (annajung) [inline _pages/open-source-day-2021-summer-contribution.md:24]: Both openstreetmap and opensuse asked the committees to reach out before posting their info on the website. Leaving it blank for now

File: _pages/open-source-day-2021-summer-contribution.md [hunk 3]
```diff
@@ -13,13 +13,27 @@ As part of the Contribution Track, you will contribute code to curated open sour
...
 
 #### Female Maintainers
 
-Coming soon
+| Project | Description | Maintainer(s) |
+| ------- | ----------- | ------------- |
+| [Tern](https://github.com/tern-tools/tern) | Tern is a software composition analysis tool and Python library that generates a Software Bill of Materials for container images and Dockerfiles. The SBoM that Tern generates will give you a layer-by-layer view of what's inside your container in a variety of formats including human-readable, JSON, HTML, SPDX and more. | [Rose Judge](https://github.com/rnjudge) and [Nisha Kumar](https://github.com/nishakm) |
+| [Truffle Shuffle](https://github.com/intuit/truffle-shuffle) | An Android data-driven, percentage-based UI Card Gallery Library. | [Katie Levy](https://github.com/katielevy1) |
+| [User Data for Fraud Prevention](https://github.com/intuit/user-data-for-fraud-prevention)| Simple npm package with a utility to collect data from the browser required for compliance with fraud prevention APIs. | [Susmitha Kodamarthi](https://github.com/skodamarthi) |
+| [Wildfly Security](https://github.com/wildfly-security/wildfly-elytron) | TBA | TBA |
```
Comment (annajung) [inline _pages/open-source-day-2021-summer-contribution.md:36]: wildfly security asked that the committees reach out before posting their info on the website. Leaving it blank for now

Comment (annajung) [reply _pages/open-source-day-2021-summer-contribution.md]: Received a response from Wildfly and added project info

**Review by renana — APPROVED** [review]

File: _pages/open-source-day-2021-summer-contribution.md (hunk 1 above)
Comment (renana) [reply _pages/open-source-day-2021-summer-contribution.md:22]: She doesn't have one as openstack is not github based.

**Pull request merged**